pkgs/development/web/insomnium/default.nix

```
@@ -0,0 +1,80 @@
+{ pkgs, stdenv, lib, makeDesktopItem, makeWrapper, fetchurl, appimageTools
+, undmg, ... }:
```

```suggestion
{ pkgs, stdenv, lib, makeDesktopItem, makeWrapper, fetchurl, appimageTools
, undmg }:
```

Could you also drop `pkgs` here to make the dependencies more explicit? You can add `mesa` if it has been used as `pkgs.mesa` before.